scripts/travis/build.sh

```diff
@@ -60,35 +60,28 @@ fi
 
 set +x
 
-# Autodiscover the platform
-PLATFORM=$(discover_platform)
-echo "Using $PLATFORM platform"
-
 # Run this manually, since if it fails when run
 # as -workspace_status_command we don't get good output
 ./scripts/release/status.sh
 
-# append the bazel default bazelrc to travis/bazel.rc
-# for using rules provided by bazel
-# cat ~/.bazelrc >> tools/travis/bazel.rc
 ./bazel_configure.py
 
 # build heron
 T="heron build"
 start_timer "$T"
 ${UTILS}/save-logs.py "heron_build.txt" bazel\
-  --bazelrc=tools/travis/bazel.rc build --config=$PLATFORM heron/... \
+  build --config=stylecheck heron/... \
```

Comment:
Not too sure about this, but I am wondering if it is strictly necessary to run style checks on all the subsequent tests? Assuming this builds the entire Heron package, it should be sufficient to run the test in one of builds?

Reply:
Agreed. We really should only need to do it once. I think I may have tested to see if the subsequent builds would be smart enough to only check the syntax for any new files introduced in the follow-up builds. But I might have to revisit that, because I don't remember what my findings were. For now. probably safer to keep as is and we can simplify and save on Travis CI build time in a follow-up PR.